assimilation_code/modules/utilities/netcdf_utilities_mod.f90

```diff
@@ -1407,6 +1625,32 @@ end subroutine nc_put_real_4d
 ! ever want to support these - just punt and make the caller drop down
 ! into direct calls to the netcdf lib.
 
+!--------------------------------------------------------------------
+
+subroutine nc_get_short_0d(ncid, varname, varval, context, filename)
+
+integer,          intent(in)  :: ncid
+character(len=*), intent(in)  :: varname
```

Comment:
This is a mistake ... I have to add a routine for i2 as well as i4 ... stay tuned.